library/general/src/lib/cfa/sysctl.rb

```diff
@@ -128,7 +130,9 @@ def load
     # +/etc/sysctl.conf+ to avoid confusion.
     def save
       super
-      clean_old_values
+
+      # we cannot update /etc/sysctl.conf in first stage as it is on ro filesystem
+      clean_old_values if Yast::Stage.normal || Yast::Stage.cont
```

Comment:
Would it be better    ... unless YaST::Stage.initial ? Just thinking about firstboot... And I cannot remember what  hardware_probed stage is for :-/